README.md

```diff
@@ -165,7 +165,6 @@ That being said, the plugin takes into account the cases where you may have inje
 Beware of the following gotchas when using this pattern:
 
 - If you're injecting `module`, `exports`, and/or `require` as dependencies, they must be injected as string literals,
-otherwise you'll end up with things like `require('module')`.
```

Comment:
Looks like this line is still being removed. There's no need for any readme changes in this PR :)

Reply:
Indeed, this was a mistake on my side. My apologies.